### ISSUE TITLE
Multipass and bg1

### ISSUE THREAD

**Issue by danieldaluz** [joelface/bg1#23]
Hello Joel!

I would like to know if the bg1 is ready for Multi pass for 1 schedule for group 1 (better attractions) and 2 schedules for group 2 (normal attractions).

By the way, you are doing an amazing job with this project. 
Congrats!

**Comment by joelface** [comment]
Yes, you can use BG1 to book Lightning Lane Multi Pass attractions. You still have to purchase Multi Pass through the My Disney Experience app (which also requires you to select at least one LLMP attraction upon purchase), but after that you can book any remaining LLs (or modify your existing ones) through BG1.

Thanks for the kind words!
